shell/platform/android/platform_view_android.cc

```diff
@@ -307,6 +307,12 @@ std::unique_ptr<Surface> PlatformViewAndroid::CreateRenderingSurface() {
   return android_surface_->CreateGPUSurface();
 }
 
+// |PlatformView|
+std::shared_ptr<ExternalViewEmbedder>
+PlatformViewAndroid::CreateExternalViewEmbedder() {
+  return external_view_embedder_;
+}
+
```

Comment:
The name of this method is CreateExternalViewEmbedder, but this method is not creating one, it's just returning one that is already created.  We should change the name of the method, or change the implementation.

Reply:
@gaaclarke I agree with this, please look at https://github.com/flutter/engine/pull/22214#issuecomment-719812222. I will rename this method to `GetExternalViewEmbedder` once https://github.com/flutter/engine/pull/22272 lands.

Reply:
Edit: I agree with your https://github.com/flutter/engine/pull/22214#issuecomment-722552371. Once we don't have to share the external view embedder with surfaces, I will make this a unique_ptr and make rasterizer the sole owner. `CreateExternalViewEmbedder` makes sense in that world for sure!